README.md

```diff
@@ -224,6 +225,44 @@ Additionally to all the methods declared in the [Parser-API](https://github.com/
 - `jsonPath()` which returns the JSON Path of the given object.
 - `meta()` which returns the metadata of a given object, like a parsed AsyncAPI Document.
 
+## Spectral rulesets
+
+Validation of AsyncAPI documents in ParserJS is powered by [Spectral](https://github.com/stoplightio/spectral). For this reason, it is possible to use own rulesets/rules, or overwrite existing ones, by passing `ruleset` option to the Parser instance:
```

Comment:
```suggestion
Validation of AsyncAPI documents in ParserJS is powered by [Spectral](https://github.com/stoplightio/spectral). For this reason, it is possible to use your own rulesets/rules, or overwrite existing ones, bypassing the `ruleset` option to the Parser instance:
```

Reply:
@alequetzalli Thanks!

`bypassing` means that something will be missed, not passed, yes? I want to wrote, that something need to be passed 😄

Reply:
hehe 😜  LMGTFY 

<img width="786" alt="Screen Shot 2023-02-13 at 6 44 05 PM" src="https://user-images.githubusercontent.com/19964402/218625503-400f8b06-a516-4dcc-9f3a-6890f1f2ff06.png">

And ah, ok! got it... then let me change that edit 😄✌🏽

Reply:
@alequetzalli Thanks! ❤️